src/ng/sniffer.js

```diff
@@ -17,6 +17,8 @@
 function $SnifferProvider() {
   this.$get = ['$window', '$document', function($window, $document) {
     var eventSupport = {},
+        isChromePackagedApp = $window.chrome && $window.chrome.app && $window.chrome.app.runtime,
+        hasHistoryPushState = $window.history && $window.history.pushState,
```

Comment:
Yes, we can. But what is the benefit ? We want to return `false` in Chrome Packaged Apps (and `'pushState' in window.history` returns true, although we are not allowed to access `history.pushState` or `histroy.state`).

(BTW, this fix is not correct, I need to update it.)